src/mappings.ts

```diff
@@ -41,12 +41,12 @@ export function handleSEED(event: SEEDTransferEvent): void {
   let tokenInstance = SEED.bind(event.address);
 
   let fromBalance = getSEEDBalance(tokenInstance, event.params.from);
-  if (fromBalance != null) {
+  if (fromBalance !== null) {
```

Comment:
I like to use `!= null` because it's rare that you want to differentiate between `null` and `undefined`, and `==` handles both. Not sure if you care in this case. There is an eslint rule that you can tweak to allow it if it complains as well.